sbnana/SBNAna/Vars/NueVars.cxx

```diff
@@ -144,7 +144,8 @@ const Var kRecoShower_trackWidth(
 const Var kRecoShowers_EnergyCut(
     [](const caf::SRSliceProxy* slc) -> unsigned {
       unsigned int counter(0);
-      for (auto const& shw : slc->reco.shw) {
+      for (auto const& pfp : slc->reco.pfp) {
```

Comment:
I think this is missing a check on the track score if we want to be completely backwards compatible